pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "dapla-toolbelt"
-version = "3.2.2"
```

Comment:
Given that we're completely removing some functionality like stuff related to `jupyterhub` from this library it is technically a breaking change which means we need to bump the major version.